tests/test_urfkill.py

```diff
@@ -73,11 +73,19 @@ def test_block(self):
         property_object = bus.get_object('org.freedesktop.URfkill', '/org/freedesktop/URfkill/WLAN')
         propiface = dbus.Interface(property_object, 'org.freedesktop.DBus.Properties')
         self.assertEqual(propiface.Get('org.freedesktop.URfkill.Killswitch', 'state'), 0)
-        iface.Block(1, True)
+
+        ret_block = iface.Block(1, True)
+        self.assertTrue(ret_block)
         self.assertEqual(propiface.Get('org.freedesktop.URfkill.Killswitch', 'state'), 1)
-        iface.Block(1, False)
+
+        ret_unblock = iface.Block(1, False)
+        self.assertTrue(ret_unblock)
         self.assertEqual(propiface.Get('org.freedesktop.URfkill.Killswitch', 'state'), 0)
 
+        # 99 is an unknown type to the mock, so it should return false.
+        ret_failed_block = iface.Block(99, False)
+        self.assertFalse(ret_failed_block)
```

Comment:
That illustrates the above, and is very confusing. "99" is the same type as "1" (int), so maybe you mean "interface number"? Does the real urfkill really return False there, instead of raising a D-Bus error (which would be a much better and common design). Can you please check urfkill's signature for that method?